src/main/java/pl/touk/sputnik/processor/detekt/DetektProcessor.java

```diff
@@ -35,22 +40,32 @@
 
     private final Configuration configuration;
 
+    private final ExecutorService executor = ForkJoinPool.commonPool();
+
+    private static PrintStream printStream;
```

Comment:
Would you mind moving it to instance field instead of a static one?

Reply:
Done

Reply:
Thanks